lib/matplotlib/collections.py

```diff
@@ -1966,6 +1968,34 @@ class QuadMesh(Collection):
     """
     Class for the efficient drawing of a quadrilateral mesh.
 
+    A quadrilateral mesh consists of a (N, M) grid of vertices, that define
```

Comment:
(M, N)
and below
(M-1)*(N-1)

(to be consistent with the indices later)

Reply:
All these PRs about quad mesh have been a bit confusing with respect to the indices convention.  I'd propose the convention is that MxN is the number of cells in the quadmesh and the coordinate are either M+1, N+1 for flat or M,N for gouraud?

Reply:
I don't have a strong opinion wrt. +/-1 here.